tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -842,11 +842,47 @@ def _restore_collections(dest_graph, src_meta_graph_def, collection_keys):
     # Ignore other meta graphs from the input SavedModel.
     saved_model_builder.save()
 
-
 def _get_resource_handle(name, device):
   with ops.device(device):
-    return gen_trt_ops.create_trt_resource_handle(resource_name=name)
+    return gen_trt_ops.create_trt_resource_handle(resource_name=name) 
+
+def _remove_native_segment(input_func):
```

Comment:
Shall we have an extra arg for `save` that would turn this on?

Reply:
Added `remove_native_segments` flag.